Orange/data/domain.py

```diff
@@ -4,44 +4,41 @@
 from numbers import Integral
 
 import weakref
+
 from .variable import *
 import numpy as np
 
 
 class DomainConversion:
-    """
-    Indices and functions for conversion between domains.
+    """Indices and functions for conversion between domains.
 
     Every list contains indices (instances of int) of variables in the
     source domain, or the variable's compute_value function if the source
     domain does not contain the variable.
 
-    .. attribute:: source
-
+    Attributes
+    ----------
+    source : int | Callable
```

Comment:
https://www.jetbrains.com/help/pycharm/2016.1/type-hinting-in-pycharm.html, section Legacy type sintax for docstrings. I considered PEP 484, but decided against it because PyCharm can infer from docstrings already and more annotations would pollute the code.